### ISSUE TITLE
Caveau (or secret room) vs Secret Group

### ISSUE THREAD

**Issue by serrq** [simplex-chat/simplex-chat#1040]
Many people know the trick whereby creating a secret group and locking yourself in as the only participant generates a kind of "secret room"-in fact, a group where you are alone.

What is the purpose of this?

Trivially to deposit files and pieces of sentences that, for convenience, we might use in the future.
We can look at this secret room as a vault in which personal effects are deposited.

But since this "vault" still inherits the robes and problems of its big brother (the secret group) I ask that the vault have as a difference from the group the blocked or broken connections.

Basically one cannot invite anyone inside (except oneself), this is to avoid inviting anyone by mistake.

This implementation should not take much time in terms of code development but would introduce a quality feature in SimpleX.
